lib/components/Editor/CustomExtensions/KeyboardShortcuts/helpers.js

```diff
@@ -0,0 +1,12 @@
+export const generateKeyboardShortcuts = (editor, shortcuts) => {
+  return shortcuts.reduce(
+    (keyboardShortcuts, shortcut) => ({
```

Comment:
I was thinking: Why don't we accept `shortcuts` prop as an `Object` rather than `Array` so that we can avoid the transformation here. I doesn't make much difference to the users also avoid unwanted keys as well. `shortcuts` can be in the format
```
shorcuts={{'Mod-Enter': (html, editor) => {}, 'Mod-Space': (html, editor) => {} }
```